src/_provider/metaOverviewAbstract.ts

```diff
@@ -165,11 +173,14 @@ export abstract class MetaOverviewAbstract {
     return this.meta;
   }
 
-  cacheObj: any = undefined;
+  cacheObj?: Cache = undefined;
 
   getCache() {
     if (this.cacheObj) return this.cacheObj;
-    this.cacheObj = new Cache(`v3/${this.url}`, 5 * 24 * 60 * 60 * 1000);
+    this.cacheObj = new Cache(
+      `v3/${api.storage.lang('locale')}/${this.url}}`,
```

Comment:
As the locale is in the cache key, meaning a different locale allways results in a new entry. This means that the comparision for a changed locale will never trigger. So the change could be removed again.

Reply:
True, got up today and thought the same.